src/sidebar/components/test/SearchInput-test.js

```diff
@@ -103,6 +117,171 @@ describe('SearchInput', () => {
+          bubbles: true,
+          cancelable: true,
+          key: 'K',
+          ctrlKey: true,
+        })
+      );
+
+      assert.equal(document.activeElement, searchInputEl);
+    });
+
+    it('focuses search input for Mac OS when "Cmd-K" is pressed outside of the component element', () => {
+      fakeIsMacOS.returns(true);
+      const wrapper = createSearchInput();
+      const searchInputEl = wrapper.find('input').getDOMNode();
+
+      document.body.dispatchEvent(
+        new KeyboardEvent('keydown', {
+          bubbles: true,
+          cancelable: true,
+          key: 'K',
+          metaKey: true,
+        })
+      );
+
+      assert.equal(document.activeElement, searchInputEl);
+    });
+
+    it('restores previous focus when focused and "Escape" key pressed', () => {
+      const button = document.createElement('button');
+      button.id = 'a-button';
+      container.append(button);
+
+      const wrapper = createSearchInput();
+
+      const inputEl = wrapper.find('input').getDOMNode();
+      const buttonEl = document.querySelector('#a-button');
+      buttonEl.focus();
+
+      buttonEl.dispatchEvent(
+        new KeyboardEvent('keydown', {
+          bubbles: true,
+          cancelable: true,
+          key: '/',
+        })
+      );
+
+      assert.equal(
+        document.activeElement,
+        inputEl,
+        'focus is moved from button to input'
+      );
+
+      inputEl.dispatchEvent(
+        new KeyboardEvent('keydown', {
+          bubbles: true,
+          cancelable: true,
+          key: 'Escape',
+        })
+      );
+
+      assert.equal(
+        document.activeElement,
+        buttonEl,
+        'focus has been restored to the button'
+      );
+    });
+
+    ['textarea', 'input'].forEach(elementName => {
+      it('does not steal focus when "/" pressed if user is in an input field', () => {
+        const input = document.createElement(elementName);
+        input.id = 'an-input';
+        container.append(input);
+
+        createSearchInput();
+
+        const inputEl = document.querySelector('#an-input');
+
+        inputEl.focus();
+
+        assert.equal(document.activeElement, inputEl);
+
+        inputEl.dispatchEvent(
+          new KeyboardEvent('keydown', {
+            bubbles: true,
+            cancelable: true,
+            key: '/',
+          })
+        );
+
+        assert.equal(
+          document.activeElement,
+          inputEl,
+          'focus has not been moved to search input'
+        );
+      });
+    });
+
+    it('focuses search input if user is in an input field and presses "Ctrl-k"', () => {
+      fakeIsMacOS.returns(false);
+      const input = document.createElement('input');
+      input.id = 'an-input';
+      container.append(input);
+
+      const wrapper = createSearchInput();
+
+      const inputEl = document.querySelector('#an-input');
+      const searchInputEl = wrapper
+        .find('[data-testid="search-input"]')
```

Comment:
Some tests are using `.find('input')` and this one is using `data-testid`. It would be good to be consistent.